src/wallet/rpcwallet.cpp

```diff
@@ -1764,6 +1764,40 @@ UniValue gettransaction(const UniValue& params, bool fHelp)
     return entry;
 }
 
+UniValue abandontransaction(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp || params.size() != 1)
+        throw runtime_error(
+            "abandontransaction \"txid\"\n"
+            "\nMark in-wallet transaction <txid> as abandoned\n"
+            "This will mark this tx and all its in-wallet descendants as abandoned which will allow\n"
```

Comment:
consistency nit: all the other fHelp messages use "transaction" and "transactions" instead of "tx" and "txs".